PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -174,6 +174,58 @@ def miniAOD_customizeCommon(process):
     for idmod in electron_ids:
         setupAllVIDIdsInModule(process,idmod,setupVIDElectronSelection)
 
+    # Adding puppi jets
+    process.load('CommonTools.PileupAlgos.Puppi_cff')
+    process.patCandidates += process.puppi
```

Comment:
3be2279f from @rappoccio in the subjet PR is actually removing such star for jets, can you please add a specific line?